README.md

```diff
@@ -1,5 +1,10 @@
 # fuzzywuzzy-rs [![docs.rs badge](https://docs.rs/fuzzywuzzy/badge.svg)](https://docs.rs/fuzzywuzzy) [![crates.io badge](https://img.shields.io/crates/v/fuzzywuzzy.svg)](https://crates.io/crates/fuzzywuzzy)
```

Comment:
Perhaps this should go onto the line below? I personally prefer the look on separate lines.

Reply:
Done.